src/pages/dataTableUtilities/actions/tableActions.js

```diff
@@ -62,6 +62,26 @@ export const hideOverStocked = () => ({ type: ACTIONS.HIDE_OVER_STOCKED });
  */
 export const hideStockOut = () => ({ type: ACTIONS.HIDE_STOCK_OUT });
 
+/**
+ * Hides all rows which have a status of finalised.
+ */
+export const showFinalised = () => ({ type: ACTIONS.SHOW_FINALISED });
```

Comment:
:nit this one `showFinalised` and that one `showNotFinalised` are only called by an internal method, so don't need to be exported. There's a really minor overhead in the export, but it also declares intention - and I wondered if those guys aren't supposed to be called externally?

Reply:
I wonder this myself - I landed on "ahhh, why not" .. not sure what the best thing is. No harm in being called directly, but the intention is that the wrapper is called, for ease of use

Reply:
oh - I also hate these names if you have any other suggestions 🤣 - I didn't want to use `past` and `current`, I felt they were less explicit ?

Reply:
might as well leave them then - if ok to call directly.

I stumbled on the Past and Current names later on (as in, didn't know what they meant at first) - so prefer the `showFinalised` and `showNotFinalised` My only other suggestion would make the naming worse, so let's not go there ( including 'only' to make it clear what is being shown `showOnlyFinalised` or `showFinalisedOnly` - but I like that idea less than what you have ). also, elsewhere in mSupply we have unfinalised rather than not. but not is more logical.

so, in summary. all good as it is. thanks.